intro-HPC/ch_introduction.tex

```diff
@@ -239,9 +239,22 @@ \subsection{Batch or interactive mode?}
 of your data without having to copy your data to your desktop and back.
 %TODO: only visualization nodes should be used interactively
 
+\subsection{What are cores, processors and nodes?}
+
+In this manual, the terms core, processor and node will be frequently used,
+so it's useful to understand what they are.
+
+Modern servers, also referred to as (worker) nodes in the context of HPC,
```

Comment:
drop space before `nodes`, and use `\emph{...}` to highlight they terms (nodes, processors, cores).